bin/util.js

```diff
@@ -57,6 +57,14 @@ module.exports = {
          */
         csvParse: (list) => {
             return _.split(list, ',');
+        },
+
+        color: (value) => {
```

Comment:
This is not type `color`. It is `colorOptions` - color would be hex code, colour names, etc